pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -209,6 +217,71 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 	}
 	managementClient := n.operatorKubeClient // We assume that operator is always running on the management cluster.
 
+	if config.MWReplicaSetEnabled {
```

Comment:
this if is not necessary, we should alway sync.

Reply:
removed this.